src/components/FilterCheckbox.vue

```diff
@@ -4,22 +4,27 @@
       v-for="[idx, topic] in Object.entries(topics)"
       :key="idx"
       class="filter-options"
+      :class="{ 'grey-out': getQuestionsnumByTags(topic, category) === '0' }"
```

Comment:
I'm thinking how we can shorten the calls, we are calling getQuestionsnumByTags 5 times. Is there a way we can set a local set/variable for these?

Reply:
This is now fixed. The function calls are limited only when the state changes in pinia.